vuepress/docs/next/tutorials/solution/customer-portal.md

```diff
@@ -4,7 +4,7 @@ sidebarDepth: 2
 
 # Entando Customer Portal 
 ## Overview
-The Entando Customer Portal (CP) is a modern, self-service customer-facing application for managing subscriptions, providing organizations with a quick ready-made solution. It includes a lightweight integration to Jira Service Management to administer service tickets and implement Role Based Access Controls (RBACs).
+The Entando Customer Portal (CP) is a modern, self-service customer-facing application for managing subscriptions, providing organizations with a quick ready-made solution. It includes a lightweight integration to Jira Service Management to administer service tickets and implement role based access controls (RBACs).
```

Comment:
I hate to say it but we've lost some of the original meaning here. This is what we started with
--
The Entando Customer Portal enables an organization to quickly provide a modern, self-service customer-facing application for management of subscriptions. It includes a lightweight integration to Jira Service Management for access to service tickets and a Role Based Access Control (RBAC) design for granting users varying levels of access.
---
The major difference for me is the second sentence. The second clause is independent of Jira - the solution provides a lightweight integration AND an RBAC design. Not a lightweight integration that implements RBAC controls. 

Can we go back to the original with an update for formatting?